ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/DataItem.h

```diff
@@ -34,7 +34,7 @@ class DataItem : public QObject
 
   QString itemId() const { return m_itemId; }
   QString path() const { return m_path; }
-  bool exists() const { return m_exists; }
```

Comment:
This wasn't particularly helpful since it was manually set and nothing used it. So I opt'd to do a runtime check of the data item which made the most sense and made the function useable.